### PR TITLE
Add selection and composition events on reference page

### PR DESCRIPTION
**Pull request by MaxPRafferty**
https://github.com/facebook/react/issues/432
Note that this PR does not update the respective Korean, Italian, or Chinese translations of this page.

### CODE REVIEW

File: docs/docs/ref-05-events.md [hunk 1]
```diff
@@ -82,6 +82,22 @@ DOMDataTransfer clipboardData
 ```
 
 
+### Composition Events
+
+Event names:
+
+```
+compositionEnd compositionStart compositionUpdate
```
Comment (spicyj) [inline docs/docs/ref-05-events.md]: This should be `onCompositionEnd` etc?

**Comment by spicyj** [comment]
Thanks, just the one nit – do you mind squashing your commits too?

**Comment by MaxPRafferty** [comment]
@spicyj Thanks; added "on" to composition events and squashed commits.

**Comment by jimfb** [comment]
Thanks!

**Pull request merged**